rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -1714,8 +1714,8 @@ spu_thread::spu_thread(lv2_spu_group* group, u32 index, std::string_view name, u
 		}
 		else
 		{
-			// 0x1000 indicates falloc to allocate page with no access rights in base memory
-			ensure(vm::get(vm::spu)->falloc(vm_offset(), SPU_LS_SIZE, &shm, 0x1200));
+			// readable/writable added flags indicate falloc to allocate page with no access rights in base memory
```

Comment:
I don't see how removal of verbose flag is justified with replacing is with much more obscure logic.

Reply:
Its not obscure IMO: 0 is a convenient default value, so is R/W memory protection. So 0=RW, any set protection flag which differs from it indicates the removal of such access permission from the default value. This way allowing to explicitly specify straying away from default RW yet not writing it when using RW.

Reply:
Rewritten with more explicit approach.